spec/services/region_backfill_spec.rb

```diff
@@ -1,6 +1,8 @@
 require "rails_helper"
 
 RSpec.describe RegionBackfill, type: :model do
+  before { skip "RegionBackfill is now deprecated and should be removed eventually." }
```

Comment:
I've currently just flagged off these specs because they shouldn't need to be run anymore since regions are always created. But I'm not sure if I'm missing anything, as far as I understand, `RegionBackfill` is a kind of `regions_prep`, so it should go away with it.